package.json

```diff
@@ -124,7 +124,9 @@
                         "required": [
                             "workspace",
                             "serverAddress",
+                            "protocol",
                             "port",
+                            "rejectUnauthorized",
```

Comment:
Minor:  Is this really required, a default of false would probably be easier for the user?